frontend/__tests__/sum.test.ts

```diff
@@ -0,0 +1,21 @@
+import { sum } from "../src/sum"
+import { describe, test, expect, beforeAll } from "@jest/globals"
+import { server } from '../src/mocks/node.ts';
+
+beforeAll(() => {
+    server.listen()
```

Comment:
Should be fine to suppress Codacy errors like these in test files.

Reply:
Removed file